docs/src/main/tut/docs/http4sExamples/csrf.md

```diff
@@ -61,9 +61,8 @@ Thus, you can use it as such:
       Ok()
   }) // This endpoint now provides a user with a new csrf token.
   
-  val dummyService2: HttpService[IO] = tsecCSRF.apply(HttpService[IO] {
+  val dummyService2: HttpService[IO] = tsecCSRF.validate()(HttpService[IO] {
```

Comment:
csrf needs some mention of the `.validate()` stuff, but I can do this if you wish

Reply:
please ;-)